tests/dragonfly/replication_test.py

```diff
@@ -1869,7 +1862,6 @@ async def test_replica_reconnections_after_network_disconnect(df_factory, df_see
             task = asyncio.create_task(proxy.serve())
 
             # Wait replica to be reconnected and synchronized with master
-            await wait_for_replica_status(c_replica, status="up")
```

Comment:
Do we still need `wait_for_replica_status` method?

Reply:
nope :)

Reply:
yes we need it. it's been called for other use-cases as well.